Tests/Functional/Service/NodePropertyValidationServiceTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Neos\Ui\Tests\Functional\Service;
+
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Tests\FunctionalTestCase;
+use Neos\Flow\Validation\Validator\NotEmptyValidator;
+use Neos\Neos\Ui\Service\NodePropertyValidationService;
+
+class NodePropertyValidationServiceTest extends FunctionalTestCase
+{
+
+    /**
+     * @var NodePropertyValidationService
+     */
+    protected $nodePropertyValidationService;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $nodePropertyValidatorServiceProxy = $this->buildAccessibleProxy(NodePropertyValidationService::class);
+        $this->nodePropertyValidationService = $this->objectManager->get($nodePropertyValidatorServiceProxy);
+    }
+
+    /**
+     * @test
+     */
+    public function resolveValidator()
+    {
+        $validator = $this->nodePropertyValidationService->_call('resolveValidator', 'Neos.Neos/Validation/NotEmptyValidator', []);
+        $this->assertInstanceOf(NotEmptyValidator::class, $validator);
+    }
+
+    /**
+     * @test
+     */
+    public function validate()
```

Comment:
I'd also add a test case for validator which is missing on the server side. Would help to understand the required behavior during future refactorings. But fine to merge as is ofc.

Reply:
Thanks. Done and fixed a bug on the way.